tests/chart_tests/test_nats_jetstream.py

```diff
@@ -99,7 +99,7 @@ def test_nats_statefulset_with_jetstream_and_tls(self, kube_version):
         } in docs[10]["spec"]["template"]["spec"]["containers"][0]["volumeMounts"]
         assert {
             "name": "nats-jetstream-client-tls-volume",
-            "mountPath": "/usr/local/share/ca-certificates/release-name-jetstream-tls-certificate-client/ca.crt",
+            "mountPath": "/usr/local/share/ca-certificates/release-name-jetstream-tls-certificate-client.crt",
```

Comment:
We need a test with a custom nats.jetstreamSSLSecretName

Reply:
I missed this one - let me add test case for  custom cert as well

Reply:
@danielhoherd jetstreamSSLSecretName this seems require additional changes - can i address this in a seperate PR